net-proxy/trojan/trojan-1.10.0.ebuild

```diff
@@ -13,7 +13,7 @@ SRC_URI="https://github.com/trojan-gfw/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.g
 
 LICENSE="GPL-3+"
 SLOT="0"
-KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+KEYWORDS="amd64 ~arm ~arm64 ~x86"
```

Comment:
Are you a member of amd64 arch team? Because if not, you can't make this change. You'll have to request stabilization to net-proxy/trojan-1.10.0 from bugzilla.